goimports/runner.bash.template

```diff
@@ -3,10 +3,14 @@
 set -euo pipefail
 
 GOIMPORTS_SHORT_PATH=@@GOIMPORTS_SHORT_PATH@@
+GO_SHORT_PATH=@@GO_SHORT_PATH@@
 ARGS=@@ARGS@@
 PREFIX_DIR_PATH=@@PREFIX_DIR_PATH@@
 PREFIX_BASE_NAME=@@PREFIX_BASE_NAME@@
 
+GO_BIN_DIR=$PWD/$(dirname $GO_SHORT_PATH)
```

Comment:
Can we get the name of the directory in the rule implementation instead? I think there is a helper for that in the `paths` package. Just trying to avoid shell scripting as much as possible because it's known to blow up in unexpected ways.